acceptance/lib/puppet/acceptance/i18ndemo_utils.rb

```diff
@@ -0,0 +1,35 @@
+module Puppet
+module Acceptance
+  module I18nDemoUtils
+
+    I18NDEMO_MODULE_NAME = "eputnam-i18ndemo"
+
+    # TODO switch to puppetlabs/beaker-i18n_helper
```

Comment:
also, if you've got better ways to do this stuff or more ideas for the helper, _please_ don't hesitate to put up a PR or 10